frontend/packages/policy-editor/src/components/PolicyCardRules/PolicyRule/PolicyAccessPackages/PolicyAccordion/PolicyAccordion.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { type ReactNode, useId, useState } from 'react';
+import cn from 'classnames';
+import { StudioButton, StudioLabelAsParagraph } from '@studio/components';
+import * as StudioIcons from '@studio/icons';
```

Comment:
Can we import what we need instead of all here? 😄

Reply:
we render icons for text strings we load, until this is fixed we need all

Reply:
Can we add it as a comment to inform other developers facing this file in the future? 😄